.travis.yml

```diff
@@ -1,9 +1,12 @@
 language: python
-env:
-  matrix:
-  - TOX_ENV=py27
-  - TOX_ENV=py34
-  - TOX_ENV=py35
+matrix:
+  include:
+    - python: "2.7"
+      env: TOX_ENV=py27
+    - python: "3.4"
+      env: TOX_ENV=py34
+    - python: "3.5"
+      env: TOX_ENV=py35
```

Comment:
python 3.5 isn't available unless you do this, which was causing a 'no-interpreter-found' error in other travis builds.